packages/embed/src/types.ts

```diff
@@ -102,6 +102,8 @@ export type BillingAddressFields = {
 export type ShippingDetails = BillingDetails
 
 export type Buyer = {
+  displayName?: string
+  externalIdentifier?: string
```

Comment:
Is this correct or should be something like the following? I think it's both an optional key and optional value.
```suggestion
  displayName?: string | null
  externalIdentifier?: string | null
```

Reply:
I think if we allow people to pass `null` for those values, then yes I'd go for what you suggested (is that the case in core-api?). Otherwise, since it's an object, not passing `displayName` or passing a string satisfies that type.

Reply:
Yes the API allows `null` for all the values (`str | None`): https://github.com/gr4vy/core-api/blob/bea747f2b93afb03d6db8661c70acda0c3855678/src/api/routers/buyers/schemas.py#L233-L240